packages/react-app/src/views/ChallengeReviewView.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect } from "react";
+import axios from "axios";
+import ChallengeReviewList from "../components/ChallengeReviewList";
+
+export default function ChallengeReviewView({ serverUrl }) {
+  const [challenges, setChallenges] = React.useState([]);
+  const [isLoading, setIsLoading] = React.useState(true);
+
+  async function fetchSubmittedChallenges() {
+    setIsLoading(true);
+    console.log("getting challenges");
+    const fetchedChallenges = await axios.get(serverUrl + `challenges`, { params: { status: "SUBMITTED" } });
+    setChallenges(fetchedChallenges.data);
+    console.log(fetchedChallenges.data);
+    setIsLoading(false);
+  }
+
+  useEffect(() => {
+    fetchSubmittedChallenges();
+  }, [serverUrl]);
+
+  async function handleApprove(userAddress, challengeId, comment) {
+    console.log(`approve ${challengeId} for ${userAddress} with comment ${comment}`);
+    await axios.patch(serverUrl + `challenges`, {
+      params: { userAddress, challengeId, comment, newStatus: "ACCEPTED" },
+    });
+    await fetchSubmittedChallenges();
```

Comment:
```suggestion
    fetchSubmittedChallenges();
```
Let's take the `await` out. I don't think it's going to change anything noticeable, but it's going to unnecessarily hold the main thread while the challenges are fetched.